module/entities/helper.js

```diff
@@ -1,4 +1,943 @@
+        return new Promise(async (resolve, reject) => {
+
+            try {
+
+                let projection = [
+                    constants.schema.ENTITYTYPE,
+                    constants.schema.GROUPS
+                ];
+
+                let entiesDocs =
+                    await this.entityDocuments({
+                        _id: entityId
+                    }, projection);
+
+                let entitiesDocument = entiesDocs.data;
+                let immediateEntities = [];
+
+                if (entitiesDocument[0] &&
+                    entitiesDocument[0].groups &&
+                    Object.keys(entitiesDocument[0].groups).length > 0
+                ) {
+
+                    let getImmediateEntityTypes =
+                        await entityTypesHelper.list({
+                            name: entitiesDocument[0].entityType
+                        }, ["immediateChildrenEntityType"]
+                        );
+
+                    let immediateEntitiesIds;
+
+                    Object.keys(entitiesDocument[0].groups).forEach(entityGroup => {
+                        if (
+                            getImmediateEntityTypes[0].immediateChildrenEntityType &&
+                            getImmediateEntityTypes[0].immediateChildrenEntityType.length > 0 &&
+                            getImmediateEntityTypes[0].immediateChildrenEntityType.includes(entityGroup)
+                        ) {
+                            immediateEntitiesIds =
+                                entitiesDocument[0].groups[entityGroup];
+                        }
+                    })
+
+                    if (
+                        Array.isArray(immediateEntitiesIds) &&
+                        immediateEntitiesIds.length > 0
+                    ) {
+
+                        let searchImmediateData = await this.search(
+                            searchText,
+                            pageSize,
+                            pageNo,
+                            immediateEntitiesIds
+                        );
+
+                        immediateEntities = searchImmediateData[0];
+                    }
+                }
+
+                return resolve(immediateEntities);
+
+            } catch (error) {
+                return reject(error);
+            }
+        })
+    }
+
+    /**
+    * Entity details information.
+    * @method 
+    * @name details
+    * @param {String} entityId - _id of entity.
+    * @return {Object} - consists of entity details information. 
+    */
+
+    static details(entityId) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let entityDocument = await this.entityDocuments(
+                    {
+                        _id: entityId
+                    },
+                    "all",
+                    ["groups"]
+                );
+
+                let projection = ["metaInformation.externalId", "metaInformation.name",
```

Comment:
@rakeshSgr - Put all elements in a new line please, everywhere and not use constants schema for this as used other places in this file